iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/utils/Utils.java

```diff
@@ -57,6 +60,8 @@ public class Utils {
   private static final String DATA_REGION_GROUP = "group-0001";
   private static final String SCHEMA_REGION_GROUP = "group-0002";
   private static final CommonConfig config = CommonDescriptor.getInstance().getConfig();
+  private static final Cache<ConsensusGroupId, RaftGroupId> cache =
+      CacheBuilder.newBuilder().weakValues().build();
```

Comment:
Add the default expiration time of 5min? This will also work in the future with region migrations

Reply:
Captain, oh my captain!